pages/index.js

```diff
@@ -1,7 +1,7 @@
 export default function Home() {
   return (
-    <h1 className="text-3xl font-bold underline">
-      wallet
+    <h1 className="text-3xl font-bold">
+      Wallet
     </h1>
   )
 }
```

Comment:
Please be sure to end all files with an empty line

Reply:
ya ta